### DISCUSSION TITLE
Folder Management / Better Collection UX

### DISCUSSION THREAD

**Discussion by appinteractive** [tinacms/tinacms#3821] [Ideas]
Coming from Forestry my Pages collection view looked like this:

<img width="1183" alt="grafik" src="https://user-images.githubusercontent.com/432828/232754567-cba335c6-0a72-40bb-ac90-9c6e8c3ccaea.png">

Now with Tina it looks like this:

<img width="1039" alt="grafik" src="https://user-images.githubusercontent.com/432828/232754750-6e5181b8-40f0-47f6-8b87-bf714b1660d7.png">

Pleas a some Folder-Management, filters (when they work) are really not enough, especially when you use the folders as breadcrumbs / routing. 

Or even better, give us the ability to craft or customize the collection views by passing a component and giving some api methods like (rename, delete, edit, etc.) and building blocks like lists etc.

Also are basic metrics missing like last edit, sort etc.

**Comment by scottgallant** [comment]
We'll be delivering V1 of folder support soon (this week): we show folders in the list but can't create them.

**Comment by appinteractive** [comment]
Awesome news! @scottgallant, I think they get created when you add the path to the file name on creation or when you rename a file with path already. So what is missing for creating folder? Any issues you see with that?

**Reply by scottgallant** [reply]
Correct, that's one way to create folders but eventually we'd like to improve that (maybe click "Create > Folder")

**Reply by appinteractive** [reply]
As there is already the possibility of multiple items, either add "folder" to the end of options, separated by a line, or add a little dedicated icon. 🤔 but with search that might be a bit crowded, so option under the create button is perfect 

But for folder creation (and renaming in particular) there need to be safeguards in place! My client was able to totally destroy the routing by using spaces, large letters and special chars 🙈 but that might be for another ticket

**Reply by appinteractive** [reply]
@scottgallant how is the feature going?

**Reply by scottgallant** [reply]
@spbyrne and @kldavis4 , do you know when folders support will be going out?

**Reply by kldavis4** [reply]
@appinteractive this is now available with `beta` tag. I think we'll release in the next day or so.
